web/meetup-facilitator/src/main.jsx

```diff
@@ -1,13 +1,34 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import { GoogleOAuthProvider } from '@react-oauth/google'
-import './index.css'
-
-ReactDOM.createRoot(document.getElementById('root')).render(
-  <GoogleOAuthProvider clientId={import.meta.env.VITE_CLIENT_ID}>
-    <React.StrictMode>
-      <App />
-    </React.StrictMode>
-  </GoogleOAuthProvider>
-  )
\ No newline at end of file
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App.jsx';
+import { GoogleOAuthProvider } from '@react-oauth/google';
+import './index.css';
+
+const apiKey = import.meta.env.VITE_API_KEY
```

Comment:
Not sure if its just mine, but I thought we wanted to change this to `VITE_GOOGLE_API_KEY` in `.env`. If not, then you can ignore this.

Reply:
the recs map doesnt load for me unless this is put for some reason, beats me but I would keep it incase

Reply:
Chris is right, in the other map we're using
```
import.meta.env.VITE_GOOGLE_API_KEY
```
Maybe the reason it's not working when you change it is because you don't have this key in your `.env`?

Reply:
oh youre talking about the variable name not the code ok yea I can change it